scripts/run_performance_job.py

```diff
@@ -546,12 +547,26 @@ def run_performance_job(args: RunPerformanceJobArgs):
         ci_setup_arguments.dotnet_path = f"{wasm_bundle_dir}/dotnet"
 
     if args.dotnet_version_link is not None:
-        with urllib.request.urlopen(args.dotnet_version_link) as response:
-            values = json.loads(response.read().decode('utf-8'))
-            if "dotnet_version" in values:
-                ci_setup_arguments.dotnet_versions = [values["dotnet_version"]]
+        if args.dotnet_version_link.startswith("https"): # Version link is a proper url
+            if args.dotnet_version_link.endswith(".json"):
```

Comment:
We should raise an error for the case that this is false

Reply:
Good catch, updated.